src/physObject.ts

```diff
@@ -0,0 +1,26 @@
+export class PhysObject {
```

Comment:
This feels more like a "Sprite" class.

Reply:
Sprites are typically the graphical assets to 2d games. Since this is a class that represents the physical attributes of an object within the game world, I believe that PhysObject is more appropriate.